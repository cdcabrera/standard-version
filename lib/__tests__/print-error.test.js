const printError = require('../print-error');

describe('print-error', () => {
  it('should pass and run console methods', () => {
    const mockError = jest.fn();
    const spy = jest.spyOn(console, 'error').mockImplementation(mockError);

    printError({ silent: false });
    printError({ silent: false }, 'lorem ipsum');
    printError({ silent: false }, 'dolor sit', { color: 'green' });
    expect(mockError.mock.calls).toMatchSnapshot('console messages');

    spy.mockClear();
  });
});
