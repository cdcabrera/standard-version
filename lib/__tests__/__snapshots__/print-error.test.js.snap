// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`print-error should pass and run console methods: console messages 1`] = `
Array [
  Array [
    "<red>undefined</red>",
  ],
  Array [
    "<red>lorem ipsum</red>",
  ],
  Array [
    "<green>dolor sit</green>",
  ],
]
`;
